src/Sylius/Behat/Page/SymfonyPage.php

```
@@ -21,7 +21,32 @@ abstract class SymfonyPage extends BaseSymfonyPage implements SymfonyPageInterfa
 {
     protected function getElement(string $name, array $parameters = []): NodeElement
     {
-        DriverHelper::waitForPageToLoad($this->getSession());
+        // Strategy: Fast path with lazy retry - all in one place!
+        // Try 1: Get element immediately (0ms wait) - works when page is stable ✅
+        // Try 2: If fails, wait for Live Components and retry
+        // Try 3: If still fails, give extra 300ms and final attempt
+
+        $lastException = null;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$lastException'. (undefined)

(UnusedLocalVariable)